client/src/main/java/com/tencent/rss/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -223,6 +225,22 @@ public void registerCoordinators(String coordinators) {
     coordinatorClients.addAll(clients);
   }
 
+  @Override
+  public Map<String, String> fetchClientConf(int timeoutMs) {
+    RssFetchClientConfResponse response =
+        new RssFetchClientConfResponse(ResponseStatusCode.INTERNAL_ERROR, "Empty coordinator clients");
+    for (CoordinatorClient coordinatorClient : coordinatorClients) {
+      response = coordinatorClient.fetchClientConf(new RssFetchClientConfRequest(timeoutMs));
+      if (response.getStatusCode() == ResponseStatusCode.SUCCESS) {
+        LOG.info("Success to get conf from {}", coordinatorClient.getDesc());
+        break;
+      } else {
+        LOG.warn("Fail to get conf from {}", coordinatorClient.getDesc());
+      }
+    }
+    return response.getClientConf();
```

Comment:
Is it possible that the response is null value?

Reply:
No, empty map.